UPGRADE.md

```diff
@@ -2,6 +2,18 @@
 
 ## dev-develop
 
+### Media Format Cache Public Folder
+
+Media `public_folder` parameter not longer exists.  
```

Comment:
The `sulu_media.format_cache.public_folder` parameter does not longer exist, and therefore has to be removed from the configuration.